File: src/C#/BehavioralPatterns/Command/FastFoodRestaurant/Commands/ModifyMenuItemCommand.cs
using FastFoodRestaurant.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FastFoodRestaurant.Commands
{
    public class ModifyMenuItemCommand : OrderCommand
    {
        public override void Execute(List<MenuItem> fastFoodOrderMenuItems, MenuItem menuItem)
        {
            var menuItemToModify = fastFoodOrderMenuItems.Where(mi => mi.Name == menuItem.Name).First();
            menuItemToModify.Price = menuItem.Price;
            menuItemToModify.Amount = menuItem.Amount;
        }
    }
}

File: src/JavaScript/BehavioralPatterns/Command/FastFoodRestaurant/commands/modifyMenuItemCommand.js

const OrderCommand = require('./orderCommand');

class ModifyMenuItemCommand extends OrderCommand {
    execute(fastFoodOrderMenuItems, menuItem) {
        const menuItemToModify = fastFoodOrderMenuItems.find(mi => mi.name === menuItem.name);

        if (menuItemToModify) {
            menuItemToModify.price = menuItem.price;
            menuItemToModify.amount = menuItem.amount;
        }
    }
}

module.exports = ModifyMenuItemCommand;